.github/workflows/examples-nrfconnect.yaml

```diff
@@ -39,6 +39,8 @@ jobs:
               with:
                   fetch-depth: 0
                   submodules: true
+            - name: Update nRF Connect SDK revision to the currently recommended.
```

Comment:
Suggestion:
`Update nRF Connect SDK revision to the latest stable development version.`

Reply:
Honestly I'm not sure if changing "recommended version/revision" to the more verbose one in a code and workflows is a good idea, as I think that it should be as short and clear as possible. IMO developers from other companies will not really care what exactly is a current version, but they would like to know that they can use it safely, because it is something that we support and recommend.